src/test/java/org/stratumproject/fabric/tna/behaviour/upf/FabricUpfProgrammableTest.java

```diff
@@ -351,6 +336,24 @@ public void testSliceMeter() throws Exception {
         assertTrue(upfProgrammable.readAll(UpfEntityType.SLICE_METER).isEmpty());
     }
 
+    @Test
+    public void testCounter() throws Exception {
```

Comment:
should we start with a nz counter in order to verify the reset behavior ?

Reply:
If not - it would be good to add a comment

Reply:
Apply API allows to do a generic modify of the counter, not just a reset.

Reply:
now looks better